st2actions/BUILD

```diff
@@ -0,0 +1,10 @@
+st2_component_python_distribution(
+    component_name="st2actions",
+    scripts=[
+        "bin/st2actionrunner",
+        "bin/st2notifier",
+        "bin/st2workflowengine",
+        "bin/st2scheduler",
+        "bin/runners.sh:shell",  # used by service files installed by st2-packaging
+    ],
```

Comment:
https://github.com/StackStorm/st2/blob/f1159344e0ca1630de13e88042589c5e472de2a0/st2actions/setup.py#L49-L54

Plus I noticed that st2-packaging needs runners.sh, so I included that as well.